docs/cookbook/frontend/map.rst.inc

```diff
@@ -1,3 +1,2 @@
 * :doc:`/cookbook/frontend/admin-js-and-css`
 * :doc:`/cookbook/frontend/google-analytics`
-* :doc:`/cookbook/frontend/webpack`
```

Comment:
You need also to remove this file from `docs/cookbook/index.rst`

Reply:
I've missed this one in `index.rst` as it was `frontend/webpack` (without `/cookbook`) :D.